server/src/main/java/org/apache/druid/server/initialization/jetty/JettyRequestLog.java

```diff
@@ -41,6 +45,20 @@ public void log(Request request, Response response)
           request.getHttpURI().toString(),
           request.getProtocol()
       );
+    } else if ((HttpMethod.POST.equals(request.getMethod())
```

Comment:
This is going to behave differently depending on whether debug logging is enabled or not. If debug is enabled, this code path won't execute, and _less_ information will get logged, and potentially at a lower level. This behavior doesn't really make sense.

I'd suggest writing out what behavior change you're suggesting and why, so we can discuss that along with the code change.

Reply:
Hi @gianm  Thanks for taking a looking into my PR.  I completely agree with you when the debug enabled , new portion will not get executed.  Normally in production cluster the debug mode may not have been enabled, The intent of the change is to identify the state change of the data such as Insert, Update, Delete (Also why, what, when)  on the data so that it can help debug and correlate issues related to data, configuration changes etc. 

The intention of adding user was to identify which user but it may not be required and I can set it only the elements as in debug mode.  This way we can maintain uniformity and backward compatibility to avoid any log parsing issues due to this change.  Please let me know what is your opinion.

Reply:
Updated the code to be compatible with debug enabled logging.